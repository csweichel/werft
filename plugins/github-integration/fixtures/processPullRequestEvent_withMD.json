{
    "Event": {
        "action": "edited",
        "number": 27,
        "pull_request": {
            "url": "https://api.github.com/repos/csweichel/test-repo/pulls/27",
            "id": 684357937,
            "node_id": "MDExOlB1bGxSZXF1ZXN0Njg0MzU3OTM3",
            "html_url": "https://github.com/csweichel/test-repo/pull/27",
            "diff_url": "https://github.com/csweichel/test-repo/pull/27.diff",
            "patch_url": "https://github.com/csweichel/test-repo/pull/27.patch",
            "issue_url": "https://api.github.com/repos/csweichel/test-repo/issues/27",
            "number": 27,
            "state": "open",
            "locked": false,
            "title": "Create .gitpod",
            "user": {
                "login": "csweichel",
                "id": 3210701,
                "node_id": "MDQ6VXNlcjMyMTA3MDE=",
                "avatar_url": "https://avatars.githubusercontent.com/u/3210701?v=4",
                "gravatar_id": "",
                "url": "https://api.github.com/users/csweichel",
                "html_url": "https://github.com/csweichel",
                "followers_url": "https://api.github.com/users/csweichel/followers",
                "following_url": "https://api.github.com/users/csweichel/following{/other_user}",
                "gists_url": "https://api.github.com/users/csweichel/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/csweichel/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/csweichel/subscriptions",
                "organizations_url": "https://api.github.com/users/csweichel/orgs",
                "repos_url": "https://api.github.com/users/csweichel/repos",
                "events_url": "https://api.github.com/users/csweichel/events{/privacy}",
                "received_events_url": "https://api.github.com/users/csweichel/received_events",
                "type": "User",
                "site_admin": false
            },
            "body": "- [x] /werft with-preview",
            "created_at": "2021-07-06T13:03:07Z",
            "updated_at": "2022-06-01T19:37:58Z",
            "closed_at": null,
            "merged_at": null,
            "merge_commit_sha": "d4c5acc3758aae17046489e3c4dae5c7f44f566b",
            "assignee": null,
            "assignees": [],
            "requested_reviewers": [],
            "requested_teams": [],
            "labels": [],
            "milestone": null,
            "draft": false,
            "commits_url": "https://api.github.com/repos/csweichel/test-repo/pulls/27/commits",
            "review_comments_url": "https://api.github.com/repos/csweichel/test-repo/pulls/27/comments",
            "review_comment_url": "https://api.github.com/repos/csweichel/test-repo/pulls/comments{/number}",
            "comments_url": "https://api.github.com/repos/csweichel/test-repo/issues/27/comments",
            "statuses_url": "https://api.github.com/repos/csweichel/test-repo/statuses/eed6dfe4dad7e3519acf52679b9880c5d2a2afbf",
            "head": {
                "label": "csweichel:cannot-init",
                "ref": "cannot-init",
                "sha": "eed6dfe4dad7e3519acf52679b9880c5d2a2afbf",
                "user": {
                    "login": "csweichel",
                    "id": 3210701,
                    "node_id": "MDQ6VXNlcjMyMTA3MDE=",
                    "avatar_url": "https://avatars.githubusercontent.com/u/3210701?v=4",
                    "gravatar_id": "",
                    "url": "https://api.github.com/users/csweichel",
                    "html_url": "https://github.com/csweichel",
                    "followers_url": "https://api.github.com/users/csweichel/followers",
                    "following_url": "https://api.github.com/users/csweichel/following{/other_user}",
                    "gists_url": "https://api.github.com/users/csweichel/gists{/gist_id}",
                    "starred_url": "https://api.github.com/users/csweichel/starred{/owner}{/repo}",
                    "subscriptions_url": "https://api.github.com/users/csweichel/subscriptions",
                    "organizations_url": "https://api.github.com/users/csweichel/orgs",
                    "repos_url": "https://api.github.com/users/csweichel/repos",
                    "events_url": "https://api.github.com/users/csweichel/events{/privacy}",
                    "received_events_url": "https://api.github.com/users/csweichel/received_events",
                    "type": "User",
                    "site_admin": false
                },
                "repo": {
                    "id": 165038235,
                    "node_id": "MDEwOlJlcG9zaXRvcnkxNjUwMzgyMzU=",
                    "name": "test-repo",
                    "full_name": "csweichel/test-repo",
                    "private": false,
                    "owner": {
                        "login": "csweichel",
                        "id": 3210701,
                        "node_id": "MDQ6VXNlcjMyMTA3MDE=",
                        "avatar_url": "https://avatars.githubusercontent.com/u/3210701?v=4",
                        "gravatar_id": "",
                        "url": "https://api.github.com/users/csweichel",
                        "html_url": "https://github.com/csweichel",
                        "followers_url": "https://api.github.com/users/csweichel/followers",
                        "following_url": "https://api.github.com/users/csweichel/following{/other_user}",
                        "gists_url": "https://api.github.com/users/csweichel/gists{/gist_id}",
                        "starred_url": "https://api.github.com/users/csweichel/starred{/owner}{/repo}",
                        "subscriptions_url": "https://api.github.com/users/csweichel/subscriptions",
                        "organizations_url": "https://api.github.com/users/csweichel/orgs",
                        "repos_url": "https://api.github.com/users/csweichel/repos",
                        "events_url": "https://api.github.com/users/csweichel/events{/privacy}",
                        "received_events_url": "https://api.github.com/users/csweichel/received_events",
                        "type": "User",
                        "site_admin": false
                    },
                    "html_url": "https://github.com/csweichel/test-repo",
                    "description": "This is just a test repo to play with GitHub apps - nothing to see here",
                    "fork": false,
                    "url": "https://api.github.com/repos/csweichel/test-repo",
                    "forks_url": "https://api.github.com/repos/csweichel/test-repo/forks",
                    "keys_url": "https://api.github.com/repos/csweichel/test-repo/keys{/key_id}",
                    "collaborators_url": "https://api.github.com/repos/csweichel/test-repo/collaborators{/collaborator}",
                    "teams_url": "https://api.github.com/repos/csweichel/test-repo/teams",
                    "hooks_url": "https://api.github.com/repos/csweichel/test-repo/hooks",
                    "issue_events_url": "https://api.github.com/repos/csweichel/test-repo/issues/events{/number}",
                    "events_url": "https://api.github.com/repos/csweichel/test-repo/events",
                    "assignees_url": "https://api.github.com/repos/csweichel/test-repo/assignees{/user}",
                    "branches_url": "https://api.github.com/repos/csweichel/test-repo/branches{/branch}",
                    "tags_url": "https://api.github.com/repos/csweichel/test-repo/tags",
                    "blobs_url": "https://api.github.com/repos/csweichel/test-repo/git/blobs{/sha}",
                    "git_tags_url": "https://api.github.com/repos/csweichel/test-repo/git/tags{/sha}",
                    "git_refs_url": "https://api.github.com/repos/csweichel/test-repo/git/refs{/sha}",
                    "trees_url": "https://api.github.com/repos/csweichel/test-repo/git/trees{/sha}",
                    "statuses_url": "https://api.github.com/repos/csweichel/test-repo/statuses/{sha}",
                    "languages_url": "https://api.github.com/repos/csweichel/test-repo/languages",
                    "stargazers_url": "https://api.github.com/repos/csweichel/test-repo/stargazers",
                    "contributors_url": "https://api.github.com/repos/csweichel/test-repo/contributors",
                    "subscribers_url": "https://api.github.com/repos/csweichel/test-repo/subscribers",
                    "subscription_url": "https://api.github.com/repos/csweichel/test-repo/subscription",
                    "commits_url": "https://api.github.com/repos/csweichel/test-repo/commits{/sha}",
                    "git_commits_url": "https://api.github.com/repos/csweichel/test-repo/git/commits{/sha}",
                    "comments_url": "https://api.github.com/repos/csweichel/test-repo/comments{/number}",
                    "issue_comment_url": "https://api.github.com/repos/csweichel/test-repo/issues/comments{/number}",
                    "contents_url": "https://api.github.com/repos/csweichel/test-repo/contents/{+path}",
                    "compare_url": "https://api.github.com/repos/csweichel/test-repo/compare/{base}...{head}",
                    "merges_url": "https://api.github.com/repos/csweichel/test-repo/merges",
                    "archive_url": "https://api.github.com/repos/csweichel/test-repo/{archive_format}{/ref}",
                    "downloads_url": "https://api.github.com/repos/csweichel/test-repo/downloads",
                    "issues_url": "https://api.github.com/repos/csweichel/test-repo/issues{/number}",
                    "pulls_url": "https://api.github.com/repos/csweichel/test-repo/pulls{/number}",
                    "milestones_url": "https://api.github.com/repos/csweichel/test-repo/milestones{/number}",
                    "notifications_url": "https://api.github.com/repos/csweichel/test-repo/notifications{?since,all,participating}",
                    "labels_url": "https://api.github.com/repos/csweichel/test-repo/labels{/name}",
                    "releases_url": "https://api.github.com/repos/csweichel/test-repo/releases{/id}",
                    "deployments_url": "https://api.github.com/repos/csweichel/test-repo/deployments",
                    "created_at": "2019-01-10T10:17:50Z",
                    "updated_at": "2022-02-20T14:29:49Z",
                    "pushed_at": "2022-05-10T17:36:43Z",
                    "git_url": "git://github.com/csweichel/test-repo.git",
                    "ssh_url": "git@github.com:csweichel/test-repo.git",
                    "clone_url": "https://github.com/csweichel/test-repo.git",
                    "svn_url": "https://github.com/csweichel/test-repo",
                    "homepage": null,
                    "size": 119,
                    "stargazers_count": 0,
                    "watchers_count": 0,
                    "language": null,
                    "has_issues": true,
                    "has_projects": true,
                    "has_downloads": true,
                    "has_wiki": true,
                    "has_pages": true,
                    "forks_count": 1,
                    "mirror_url": null,
                    "archived": false,
                    "disabled": false,
                    "open_issues_count": 9,
                    "license": null,
                    "allow_forking": true,
                    "is_template": false,
                    "topics": [],
                    "visibility": "public",
                    "forks": 1,
                    "open_issues": 9,
                    "watchers": 0,
                    "default_branch": "master",
                    "allow_squash_merge": true,
                    "allow_merge_commit": true,
                    "allow_rebase_merge": true,
                    "allow_auto_merge": false,
                    "delete_branch_on_merge": false,
                    "allow_update_branch": false
                }
            },
            "base": {
                "label": "csweichel:master",
                "ref": "master",
                "sha": "2079d8c3729c1ccdac32d15f06d2b2119b315570",
                "user": {
                    "login": "csweichel",
                    "id": 3210701,
                    "node_id": "MDQ6VXNlcjMyMTA3MDE=",
                    "avatar_url": "https://avatars.githubusercontent.com/u/3210701?v=4",
                    "gravatar_id": "",
                    "url": "https://api.github.com/users/csweichel",
                    "html_url": "https://github.com/csweichel",
                    "followers_url": "https://api.github.com/users/csweichel/followers",
                    "following_url": "https://api.github.com/users/csweichel/following{/other_user}",
                    "gists_url": "https://api.github.com/users/csweichel/gists{/gist_id}",
                    "starred_url": "https://api.github.com/users/csweichel/starred{/owner}{/repo}",
                    "subscriptions_url": "https://api.github.com/users/csweichel/subscriptions",
                    "organizations_url": "https://api.github.com/users/csweichel/orgs",
                    "repos_url": "https://api.github.com/users/csweichel/repos",
                    "events_url": "https://api.github.com/users/csweichel/events{/privacy}",
                    "received_events_url": "https://api.github.com/users/csweichel/received_events",
                    "type": "User",
                    "site_admin": false
                },
                "repo": {
                    "id": 165038235,
                    "node_id": "MDEwOlJlcG9zaXRvcnkxNjUwMzgyMzU=",
                    "name": "test-repo",
                    "full_name": "csweichel/test-repo",
                    "private": false,
                    "owner": {
                        "login": "csweichel",
                        "id": 3210701,
                        "node_id": "MDQ6VXNlcjMyMTA3MDE=",
                        "avatar_url": "https://avatars.githubusercontent.com/u/3210701?v=4",
                        "gravatar_id": "",
                        "url": "https://api.github.com/users/csweichel",
                        "html_url": "https://github.com/csweichel",
                        "followers_url": "https://api.github.com/users/csweichel/followers",
                        "following_url": "https://api.github.com/users/csweichel/following{/other_user}",
                        "gists_url": "https://api.github.com/users/csweichel/gists{/gist_id}",
                        "starred_url": "https://api.github.com/users/csweichel/starred{/owner}{/repo}",
                        "subscriptions_url": "https://api.github.com/users/csweichel/subscriptions",
                        "organizations_url": "https://api.github.com/users/csweichel/orgs",
                        "repos_url": "https://api.github.com/users/csweichel/repos",
                        "events_url": "https://api.github.com/users/csweichel/events{/privacy}",
                        "received_events_url": "https://api.github.com/users/csweichel/received_events",
                        "type": "User",
                        "site_admin": false
                    },
                    "html_url": "https://github.com/csweichel/test-repo",
                    "description": "This is just a test repo to play with GitHub apps - nothing to see here",
                    "fork": false,
                    "url": "https://api.github.com/repos/csweichel/test-repo",
                    "forks_url": "https://api.github.com/repos/csweichel/test-repo/forks",
                    "keys_url": "https://api.github.com/repos/csweichel/test-repo/keys{/key_id}",
                    "collaborators_url": "https://api.github.com/repos/csweichel/test-repo/collaborators{/collaborator}",
                    "teams_url": "https://api.github.com/repos/csweichel/test-repo/teams",
                    "hooks_url": "https://api.github.com/repos/csweichel/test-repo/hooks",
                    "issue_events_url": "https://api.github.com/repos/csweichel/test-repo/issues/events{/number}",
                    "events_url": "https://api.github.com/repos/csweichel/test-repo/events",
                    "assignees_url": "https://api.github.com/repos/csweichel/test-repo/assignees{/user}",
                    "branches_url": "https://api.github.com/repos/csweichel/test-repo/branches{/branch}",
                    "tags_url": "https://api.github.com/repos/csweichel/test-repo/tags",
                    "blobs_url": "https://api.github.com/repos/csweichel/test-repo/git/blobs{/sha}",
                    "git_tags_url": "https://api.github.com/repos/csweichel/test-repo/git/tags{/sha}",
                    "git_refs_url": "https://api.github.com/repos/csweichel/test-repo/git/refs{/sha}",
                    "trees_url": "https://api.github.com/repos/csweichel/test-repo/git/trees{/sha}",
                    "statuses_url": "https://api.github.com/repos/csweichel/test-repo/statuses/{sha}",
                    "languages_url": "https://api.github.com/repos/csweichel/test-repo/languages",
                    "stargazers_url": "https://api.github.com/repos/csweichel/test-repo/stargazers",
                    "contributors_url": "https://api.github.com/repos/csweichel/test-repo/contributors",
                    "subscribers_url": "https://api.github.com/repos/csweichel/test-repo/subscribers",
                    "subscription_url": "https://api.github.com/repos/csweichel/test-repo/subscription",
                    "commits_url": "https://api.github.com/repos/csweichel/test-repo/commits{/sha}",
                    "git_commits_url": "https://api.github.com/repos/csweichel/test-repo/git/commits{/sha}",
                    "comments_url": "https://api.github.com/repos/csweichel/test-repo/comments{/number}",
                    "issue_comment_url": "https://api.github.com/repos/csweichel/test-repo/issues/comments{/number}",
                    "contents_url": "https://api.github.com/repos/csweichel/test-repo/contents/{+path}",
                    "compare_url": "https://api.github.com/repos/csweichel/test-repo/compare/{base}...{head}",
                    "merges_url": "https://api.github.com/repos/csweichel/test-repo/merges",
                    "archive_url": "https://api.github.com/repos/csweichel/test-repo/{archive_format}{/ref}",
                    "downloads_url": "https://api.github.com/repos/csweichel/test-repo/downloads",
                    "issues_url": "https://api.github.com/repos/csweichel/test-repo/issues{/number}",
                    "pulls_url": "https://api.github.com/repos/csweichel/test-repo/pulls{/number}",
                    "milestones_url": "https://api.github.com/repos/csweichel/test-repo/milestones{/number}",
                    "notifications_url": "https://api.github.com/repos/csweichel/test-repo/notifications{?since,all,participating}",
                    "labels_url": "https://api.github.com/repos/csweichel/test-repo/labels{/name}",
                    "releases_url": "https://api.github.com/repos/csweichel/test-repo/releases{/id}",
                    "deployments_url": "https://api.github.com/repos/csweichel/test-repo/deployments",
                    "created_at": "2019-01-10T10:17:50Z",
                    "updated_at": "2022-02-20T14:29:49Z",
                    "pushed_at": "2022-05-10T17:36:43Z",
                    "git_url": "git://github.com/csweichel/test-repo.git",
                    "ssh_url": "git@github.com:csweichel/test-repo.git",
                    "clone_url": "https://github.com/csweichel/test-repo.git",
                    "svn_url": "https://github.com/csweichel/test-repo",
                    "homepage": null,
                    "size": 119,
                    "stargazers_count": 0,
                    "watchers_count": 0,
                    "language": null,
                    "has_issues": true,
                    "has_projects": true,
                    "has_downloads": true,
                    "has_wiki": true,
                    "has_pages": true,
                    "forks_count": 1,
                    "mirror_url": null,
                    "archived": false,
                    "disabled": false,
                    "open_issues_count": 9,
                    "license": null,
                    "allow_forking": true,
                    "is_template": false,
                    "topics": [],
                    "visibility": "public",
                    "forks": 1,
                    "open_issues": 9,
                    "watchers": 0,
                    "default_branch": "master",
                    "allow_squash_merge": true,
                    "allow_merge_commit": true,
                    "allow_rebase_merge": true,
                    "allow_auto_merge": false,
                    "delete_branch_on_merge": false,
                    "allow_update_branch": false
                }
            },
            "_links": {
                "self": {
                    "href": "https://api.github.com/repos/csweichel/test-repo/pulls/27"
                },
                "html": {
                    "href": "https://github.com/csweichel/test-repo/pull/27"
                },
                "issue": {
                    "href": "https://api.github.com/repos/csweichel/test-repo/issues/27"
                },
                "comments": {
                    "href": "https://api.github.com/repos/csweichel/test-repo/issues/27/comments"
                },
                "review_comments": {
                    "href": "https://api.github.com/repos/csweichel/test-repo/pulls/27/comments"
                },
                "review_comment": {
                    "href": "https://api.github.com/repos/csweichel/test-repo/pulls/comments{/number}"
                },
                "commits": {
                    "href": "https://api.github.com/repos/csweichel/test-repo/pulls/27/commits"
                },
                "statuses": {
                    "href": "https://api.github.com/repos/csweichel/test-repo/statuses/eed6dfe4dad7e3519acf52679b9880c5d2a2afbf"
                }
            },
            "author_association": "OWNER",
            "auto_merge": null,
            "active_lock_reason": null,
            "merged": false,
            "mergeable": true,
            "rebaseable": true,
            "mergeable_state": "unstable",
            "merged_by": null,
            "comments": 8,
            "review_comments": 0,
            "maintainer_can_modify": false,
            "commits": 1,
            "additions": 2,
            "deletions": 0,
            "changed_files": 1
        },
        "changes": {
            "body": {
                "from": "- [ ] /werft with-preview"
            }
        },
        "repository": {
            "id": 165038235,
            "node_id": "MDEwOlJlcG9zaXRvcnkxNjUwMzgyMzU=",
            "name": "test-repo",
            "full_name": "csweichel/test-repo",
            "private": false,
            "owner": {
                "login": "csweichel",
                "id": 3210701,
                "node_id": "MDQ6VXNlcjMyMTA3MDE=",
                "avatar_url": "https://avatars.githubusercontent.com/u/3210701?v=4",
                "gravatar_id": "",
                "url": "https://api.github.com/users/csweichel",
                "html_url": "https://github.com/csweichel",
                "followers_url": "https://api.github.com/users/csweichel/followers",
                "following_url": "https://api.github.com/users/csweichel/following{/other_user}",
                "gists_url": "https://api.github.com/users/csweichel/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/csweichel/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/csweichel/subscriptions",
                "organizations_url": "https://api.github.com/users/csweichel/orgs",
                "repos_url": "https://api.github.com/users/csweichel/repos",
                "events_url": "https://api.github.com/users/csweichel/events{/privacy}",
                "received_events_url": "https://api.github.com/users/csweichel/received_events",
                "type": "User",
                "site_admin": false
            },
            "html_url": "https://github.com/csweichel/test-repo",
            "description": "This is just a test repo to play with GitHub apps - nothing to see here",
            "fork": false,
            "url": "https://api.github.com/repos/csweichel/test-repo",
            "forks_url": "https://api.github.com/repos/csweichel/test-repo/forks",
            "keys_url": "https://api.github.com/repos/csweichel/test-repo/keys{/key_id}",
            "collaborators_url": "https://api.github.com/repos/csweichel/test-repo/collaborators{/collaborator}",
            "teams_url": "https://api.github.com/repos/csweichel/test-repo/teams",
            "hooks_url": "https://api.github.com/repos/csweichel/test-repo/hooks",
            "issue_events_url": "https://api.github.com/repos/csweichel/test-repo/issues/events{/number}",
            "events_url": "https://api.github.com/repos/csweichel/test-repo/events",
            "assignees_url": "https://api.github.com/repos/csweichel/test-repo/assignees{/user}",
            "branches_url": "https://api.github.com/repos/csweichel/test-repo/branches{/branch}",
            "tags_url": "https://api.github.com/repos/csweichel/test-repo/tags",
            "blobs_url": "https://api.github.com/repos/csweichel/test-repo/git/blobs{/sha}",
            "git_tags_url": "https://api.github.com/repos/csweichel/test-repo/git/tags{/sha}",
            "git_refs_url": "https://api.github.com/repos/csweichel/test-repo/git/refs{/sha}",
            "trees_url": "https://api.github.com/repos/csweichel/test-repo/git/trees{/sha}",
            "statuses_url": "https://api.github.com/repos/csweichel/test-repo/statuses/{sha}",
            "languages_url": "https://api.github.com/repos/csweichel/test-repo/languages",
            "stargazers_url": "https://api.github.com/repos/csweichel/test-repo/stargazers",
            "contributors_url": "https://api.github.com/repos/csweichel/test-repo/contributors",
            "subscribers_url": "https://api.github.com/repos/csweichel/test-repo/subscribers",
            "subscription_url": "https://api.github.com/repos/csweichel/test-repo/subscription",
            "commits_url": "https://api.github.com/repos/csweichel/test-repo/commits{/sha}",
            "git_commits_url": "https://api.github.com/repos/csweichel/test-repo/git/commits{/sha}",
            "comments_url": "https://api.github.com/repos/csweichel/test-repo/comments{/number}",
            "issue_comment_url": "https://api.github.com/repos/csweichel/test-repo/issues/comments{/number}",
            "contents_url": "https://api.github.com/repos/csweichel/test-repo/contents/{+path}",
            "compare_url": "https://api.github.com/repos/csweichel/test-repo/compare/{base}...{head}",
            "merges_url": "https://api.github.com/repos/csweichel/test-repo/merges",
            "archive_url": "https://api.github.com/repos/csweichel/test-repo/{archive_format}{/ref}",
            "downloads_url": "https://api.github.com/repos/csweichel/test-repo/downloads",
            "issues_url": "https://api.github.com/repos/csweichel/test-repo/issues{/number}",
            "pulls_url": "https://api.github.com/repos/csweichel/test-repo/pulls{/number}",
            "milestones_url": "https://api.github.com/repos/csweichel/test-repo/milestones{/number}",
            "notifications_url": "https://api.github.com/repos/csweichel/test-repo/notifications{?since,all,participating}",
            "labels_url": "https://api.github.com/repos/csweichel/test-repo/labels{/name}",
            "releases_url": "https://api.github.com/repos/csweichel/test-repo/releases{/id}",
            "deployments_url": "https://api.github.com/repos/csweichel/test-repo/deployments",
            "created_at": "2019-01-10T10:17:50Z",
            "updated_at": "2022-02-20T14:29:49Z",
            "pushed_at": "2022-05-10T17:36:43Z",
            "git_url": "git://github.com/csweichel/test-repo.git",
            "ssh_url": "git@github.com:csweichel/test-repo.git",
            "clone_url": "https://github.com/csweichel/test-repo.git",
            "svn_url": "https://github.com/csweichel/test-repo",
            "homepage": null,
            "size": 119,
            "stargazers_count": 0,
            "watchers_count": 0,
            "language": null,
            "has_issues": true,
            "has_projects": true,
            "has_downloads": true,
            "has_wiki": true,
            "has_pages": true,
            "forks_count": 1,
            "mirror_url": null,
            "archived": false,
            "disabled": false,
            "open_issues_count": 9,
            "license": null,
            "allow_forking": true,
            "is_template": false,
            "topics": [],
            "visibility": "public",
            "forks": 1,
            "open_issues": 9,
            "watchers": 0,
            "default_branch": "master"
        },
        "sender": {
            "login": "csweichel",
            "id": 3210701,
            "node_id": "MDQ6VXNlcjMyMTA3MDE=",
            "avatar_url": "https://avatars.githubusercontent.com/u/3210701?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/csweichel",
            "html_url": "https://github.com/csweichel",
            "followers_url": "https://api.github.com/users/csweichel/followers",
            "following_url": "https://api.github.com/users/csweichel/following{/other_user}",
            "gists_url": "https://api.github.com/users/csweichel/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/csweichel/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/csweichel/subscriptions",
            "organizations_url": "https://api.github.com/users/csweichel/orgs",
            "repos_url": "https://api.github.com/users/csweichel/repos",
            "events_url": "https://api.github.com/users/csweichel/events{/privacy}",
            "received_events_url": "https://api.github.com/users/csweichel/received_events",
            "type": "User",
            "site_admin": false
        },
        "installation": {
            "id": 5647067,
            "node_id": "MDIzOkludGVncmF0aW9uSW5zdGFsbGF0aW9uNTY0NzA2Nw=="
        }
    },
    "ListResponse": [
        {
            "Metadata": {
                "Repository": {
                    "Revision": "eed6dfe4dad7e3519acf52679b9880c5d2a2afbf"
                }
            }
        }
    ]
}