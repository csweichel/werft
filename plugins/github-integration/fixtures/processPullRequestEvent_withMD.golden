{
  "metadata": {
    "owner": "csweichel",
    "repository": {
      "host": "github.com",
      "owner": "csweichel",
      "repo": "test-repo",
      "ref": "refs/heads/cannot-init",
      "revision": "eed6dfe4dad7e3519acf52679b9880c5d2a2afbf",
      "defaultBranch": "master"
    },
    "trigger": "TRIGGER_MANUAL",
    "annotations": [
      {
        "key": "updateGitHubStatus",
        "value": "csweichel/test-repo"
      },
      {
        "key": "with-preview"
      }
    ]
  },
  "spec": {
    "jobPath": ""
  }
}